src/App.js

```diff
@@ -0,0 +1,25 @@
+import './css/App.css';
+import Navbar from './components/layout/Navbar';
+import Home from './components/pages/Home';
+import Deals from './components/pages/Deals';
+import Cart from './components/pages/Cart';
+//importing react router
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+function App() {
+  return (
+    // wrapping page content in the router
+    <Router>
+      <div className='App'>
+        <Navbar />
+        <Switch>
+          <Route exact path='/' component={Home} />
```

Comment:
The newest version of `react-router` has a new recommended way of creating routes via children components instead of using props so for example home route would be 

```
<Route exact path="/">
  <Home />
</Route>
```